### PR TITLE
fix: shmtu route rewrite

### PR DESCRIPTION
**Pull request by simonsmh**

### CODE REVIEW

File: docs/university.md [hunk 1]
```diff
@@ -2,11 +2,9 @@
 
 ## 上海海事大学
 
-<Route name="学术讲座" author="simonsmh" example="/shmtu/events" path="/universities/shmtu/events"/>
+<Route name="官网信息" author="simonsmh" example="/shmtu/www/events" path="/universities/shmtu/www/events" :paramsDesc="['events 为学术讲座, notes 为通知公告']"/>
```
Comment (DIYgod) [inline docs/university.md]: path 应该为 `/shmtu/www/:type`

File: docs/university.md [hunk 2]
```diff
@@ -2,11 +2,9 @@
...
 
-<Route name="通知公告" author="simonsmh" example="/shmtu/notes" path="/universities/shmtu/notes"/>
-
-<Route name="教务信息" author="simonsmh" example="/shmtu/jwc/1" path="/universities/shmtu/jwc/:type" :paramsDesc="['1 为教务新闻, 2 为教务公告']"/>
+<Route name="教务信息" author="simonsmh" example="/shmtu/jwc/jiaowugonggao" path="/universities/shmtu/jwc/:type" :paramsDesc="['jiaowuxinwen 为教务新闻, jiaowugonggao 为教务公告']"/>
```
Comment (DIYgod) [inline docs/university.md]: path 应该为 `/shmtu/jwc/:type`

**Comment by simonsmh** [comment]
Done

**Comment by DIYgod** [comment]
Merged, thanks!

**Pull request merged**